pom.xml

```diff
@@ -99,6 +99,50 @@ The Initial Developer of the Covered Software is
         <groupId>org.sonatype.plugins</groupId>
         <artifactId>nexus-staging-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>4.1</version>
+        <configuration>
+          <defaultBasedir>${project.basedir}</defaultBasedir>
```

Comment:
Because shade plugin changes it:

https://maven.apache.org/plugins/maven-shade-plugin/shade-mojo.html#dependencyReducedPomLocation